project/settings.py

```
@@ -195,9 +189,33 @@
 }
 
+SOCIALACCOUNT_PROVIDERS = {
+  'google': {
+      'APP': {
+            'client_id': config('GOOGLE_CLIENT_ID'),
+            'secret': config('GOOGLE_CLIENT_SECRET'),
+            'key': config('GOOGLE_API_KEY'),
+      },
+      'EMAIL_AUTHENTICATION': True,
+      'FETCH_USERINFO': True,
+  }
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)